addons/core/functions/fnc_getSwSettings.sqf

```diff
@@ -65,6 +65,20 @@ if (isNil "_rc") then {
     _value set [TFAR_CODE_OFFSET, _rc];
 };
 
+private _hdo = _value select HALFDUPLEX_OVERRIDE_OFFSET;
```

Comment:
https://github.com/michail-nikolaev/task-force-arma-3-radio/blob/7c2fc03dcf659c54f0b12a1c4fc98b054b40f4bc/arma3/%40task_force_radio/addons/task_force_radio/functions/fn_onSwTangentReleased.sqf#L8
Wow. Okey.
Leave it then. I'll have to add a function for CfgWeapons later on.